docs/attribution.md

```diff
@@ -20,9 +20,7 @@ Attribution needs to "appear in a place that is reasonable to the medium or mean
 
 3DEP (formerly NED) are several public domain datasets released by USGS:
```

Comment:
Since we have multiple 3DEPs we should list those out a bit here? I'm thinking of the topobathy that is in California, for instance.

Reply:
Or that could be added to the data sources page. Yeah, probably better there since it's also next to the footprints DB download.

Reply:
Fixed in 455a9c6.